app/controllers/users_controller.rb

```diff
@@ -107,7 +107,8 @@ class UsersController < ApplicationController
 
   before_action :add_noindex_header, only: %i[show my_redirect]
 
-  allow_in_staff_writes_only_mode :admin_login, :email_login, :password_reset_update
+  allow_in_readonly_mode :admin_login
```

Comment:
Yet another part of the "fix". The `/u/admin-login` is where the "user" asks for a "backdoor" via email